components/badge/overview.md

```diff
@@ -0,0 +1,72 @@
+1. Use the `ChildContent` to add content.
+1. (optional) Customize the [appearance]({%slug badge-appearance%}) and fine-tune the [positioning and alignment]({%slug badge-position-alignment%}) of the Telerik Blazor Badge.
+
+>caption Basic configuration of the Blazor Badge
+
+````CSHTML
+<TelerikButton>
+    <TelerikBadge>12</TelerikBadge>
+    <span>Notifications</span>
+</TelerikButton>
+````
+
+## Positioning and Alignment
+
+You can control how the Badge is positioned relative to its associated container. [Read more about the Blazor Badge positioning...]({%slug badge-position-alignment%})
+
+## Badge Parameters
+
+The Blazor Badge provides various parameters that allow you to configure the component. Also check the [Badge API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikBadge) for a full list of properties.
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `ShowCutoutBorder` | `bool` | Controls whether there is a margin between the Badge and the container when they overlap. |
+| `Position` | `BadgePosition ` <br /> (`Edge`) | Specifies the position of the Badge relative to the container. [Read more about Badge positioning.]({%slug badge-position-alignment%}) |
+| `HorizontalAlign` | `BadgeHorizontalAlign ` <br /> (`End`) | Determines if the left or the right side of the Badge will touch its parent container. [Read more about Badge positioning.]({%slug badge-position-alignment%}) |
+| `VerticalAlign` | `BadgeVerticalAlign  ` <br /> (`Top`) | Determines if the Badge will touch the parent container with its top or bottom side. [Read more about Badge positioning.]({%slug badge-position-alignment%}) |
+
+### Styling and Appearance
+
+The following parameters enable you to customize the appearance of the Blazor Badge:
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | Defines the `<div class="k-badge">` CSS class rendered on the main wrapping element of the Badge component. Use it for [styling customizations]({%slug themes-override%}). |
+| `FillMode` | `Telerik.Blazor.ThemeConstants.Badge.FillMode.Solid` | Controls how the Badge is filled. |
+| `Rounded` | `Telerik.Blazor.ThemeConstants.Badge.Rounded.Full` | Defines how rounded the borders of the Badge are. |
+| `Size` | `Telerik.Blazor.ThemeConstants.Badge.Size.Medium` | Sets the size of the Badge. |
+| `ThemeColor` | `Telerik.Blazor.ThemeConstants.Badge.ThemeColor.Primary` | Adjusts the background color of the Badge. |
+
+You can find more information for customizing the Badge appearance in the [Appearance article]({%slug badge-appearance%}).
+
+## Next Steps
+
+* [Explore the Badge Positioning and Alignment Settings]({%slug badge-position-alignment%})
+* [Customize the Badge Appearance]({%slug badge-appearance%})
+* [Add a Badge to a Container that Cannot Have Children (Knowledge-Base Article)]({slug kb-badge-container-no-children})
```

Comment:
The slug syntax is invalid. Is this on purpose due to the missing KB? I would add the KB directly to this PR.

Reply:
I have not yet added the KB, it's in my to-do list for the docs.